Submitting plugin User Activity

edwink75 opened the pull request.

saade reviewed (changes requested):
Hey! Your plugin doesn't have an image, please add images as per our [Contributing guide](https://github.com/filamentphp/filamentphp.com#submitting-a-plugin). It must fit the 16:9 aspect ratio, at least 2560x1440 pixels in size, and preferably a JPEG. They are mandatory. For plugins, you could use [Beyond Code's banner generator](https://banners.beyondco.de) to generate them.

Please let us know when you've addressed these issues.

edwink75 commented:
> Hey! Your plugin doesn't have an image, please add images as per our [Contributing guide](https://github.com/filamentphp/filamentphp.com#submitting-a-plugin). It must fit the 16:9 aspect ratio, at least 2560x1440 pixels in size, and preferably a JPEG. They are mandatory. For plugins, you could use [Beyond Code's banner generator](https://banners.beyondco.de) to generate them.
> 
> Please let us know when you've addressed these issues.

JPEG added. 
All good now?

saade reviewed (changes requested):
Hey! Could you please ensure that image URLs in your plugin's `README` file are absolute URLs rather than relative links? Then we can embed them on our website correctly. Thanks!

Please let us know when you've addressed these issues.

edwink75 commented:
> Hey! Could you please ensure that image URLs in your plugin's `README` file are absolute URLs rather than relative links? Then we can embed them on our website correctly. Thanks!
> 
> Please let us know when you've addressed these issues.

I changed that just now... Hope it's okay now.

saade commented:
> > Hey! Could you please ensure that image URLs in your plugin's `README` file are absolute URLs rather than relative links? Then we can embed them on our website correctly. Thanks!
> > Please let us know when you've addressed these issues.
> 
> I changed that just now... Hope it's okay now.

The raw url is wrong, it should be:
https://raw.githubusercontent.com/edwink75/filament-user-activity/main/img/Active-Users-Table.png

edwink75 commented:
> > > Hey! Could you please ensure that image URLs in your plugin's `README` file are absolute URLs rather than relative links? Then we can embed them on our website correctly. Thanks!
> > > Please let us know when you've addressed these issues.
> > 
> > 
> > I changed that just now... Hope it's okay now.
> 
> The raw url is wrong, it should be: https://raw.githubusercontent.com/edwink75/filament-user-activity/main/img/Active-Users-Table.png

used your URL now.

edwink75 commented:
Hi Guilherme,
Could you adjust the Github link please?
https://github.com/edwink/filament-user-activity
to
https://github.com/edwink75/filament-user-activity
Thank you!
![image](https://github.com/filamentphp/filamentphp.com/assets/10741779/337526c4-3ce3-447f-9f84-d6b5fd9aa014)

saade commented:
Sure, done.
